File: 1832. Check if the Sentence Is Pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        boolean[] chars = new boolean[26];

        for (char c : sentence.toCharArray()) {
            chars[c - 'a'] = true;
        }

        for (boolean b : chars) {
            if (!b) return false;
        }

        return true;
    }
}

File: 1832. Check if the Sentence Is Pangram.go
func checkIfPangram(sentence string) bool {
    bools := [26]bool{}
    for _, c := range sentence {
        bools[c - 'a'] = true
    }

    for _, b := range bools {
        if !b {
            return false
        }
    }

    return true
}